apps/docs/docs/ref/kotlin/introduction.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: introduction
+title: Introduction
+hideTitle: true
+---
+
+<div className="flex items-start gap-6 not-prose" id="introduction">
+  <img src="/docs/img/icons/menu/reference-kotlin.svg" className="w-8 h-8 rounded" />
+  <div className="flex flex-col gap-2">
+    <h1 className="text-3xl text-scale-1200 m-0">Kotlin Client Library</h1>
+    <h2 className="text-base font-mono text-scale-1100">@supabase-community/supabase-kt</h2>
+  </div>
+</div>
+
+<div className="max-w-xl">
+This reference documents every object and method available in Supabase's Kotlin Multiplatform library, [supabase-kt](https://github.com/supabase-community/supabase-kt). You can use supabase-kt to interact with your Postgres database, listen to database changes, invoke Deno Edge Functions, build login and user management functionality, and manage large files.
+
+Supported Kotlin targets:
+    - JVM (Desktop)
```

Comment:
Isn't JVM target universal (so also server environment?) If yes, I'd remove the suffix